README.md

```diff
@@ -92,381 +92,172 @@
-import torch
+You can start by reading [our basicss](https://unify.ai/docs/ivy/demos/learn_the_basics.html) and [demos](https://unify.ai/docs/ivy/demos/examples_and_demos.html). 
 
-def mse_loss(y, target):
-    return ivy.mean((y - target)**2)
-
-jax_mse   = mse_loss(jnp.ones((5,)), jnp.ones((5,)))
-tf_mse    = mse_loss(tf.ones((5,)), tf.ones((5,)))
-np_mse    = mse_loss(np.ones((5,)), np.ones((5,)))
-torch_mse = mse_loss(torch.ones((5,)), torch.ones((5,)))
-```
+here is a good starting point for you:
 
-In the example above we show how Ivy\'s functions are compatible with
-tensors from different frameworks. This is the same for ALL Ivy
-functions. They can accept tensors from any framework and return the
-correct result.
-
-The [Ivy Stateful
-API](https://unify.ai/docs/ivy/overview/design/ivy_as_a_framework/ivy_stateful_api.html),
-on the other hand, allows you to define trainable modules and layers,
-which you can use alone or as a part of any other framework code!
-
-``` python
-import ivy
-
-
-class Regressor(ivy.Module):
-    def __init__(self, input_dim, output_dim):
-        self.input_dim = input_dim
-        self.output_dim = output_dim
-        super().__init__()
-
-    def _build(self, *args, **kwargs):
-        self.linear0 = ivy.Linear(self.input_dim, 128)
-        self.linear1 = ivy.Linear(128, self.output_dim)
-
-    def _forward(self, x):
-        x = self.linear0(x)
-        x = ivy.functional.relu(x)
-        x = self.linear1(x)
-        return x
-```
-
-If we put it all together, we\'ll have something like this. This example
-uses PyTorch as the backend, but this can easily be changed to your
-favorite frameworks, such as TensorFlow, or JAX.
-
-``` python
-import ivy
-
-
-class Regressor(ivy.Module):
-    def __init__(self, input_dim, output_dim):
-        self.input_dim = input_dim
-        self.output_dim = output_dim
-        super().__init__()
-
-    def _build(self, *args, **kwargs):
-        self.linear0 = ivy.Linear(self.input_dim, 128)
-        self.linear1 = ivy.Linear(128, self.output_dim)
-
-    def _forward(self, x):
-        x = self.linear0(x)
-        x = ivy.functional.relu(x)
-        x = self.linear1(x)
-        return x
-
-ivy.set_backend('torch')  # set backend to PyTorch (or any other backend!)
-
-model = Regressor(input_dim=1, output_dim=1)
-optimizer = ivy.Adam(0.3)
-
-n_training_examples = 2000
-noise = ivy.random.random_normal(shape=(n_training_examples, 1), mean=0, std=0.1)
-x = ivy.linspace(-6, 3, n_training_examples).reshape((n_training_examples, 1))
-y = 0.2 * x ** 2 + 0.5 * x + 0.1 + noise
+- [How to Convert your code between frameworks?](https://unify.ai/docs/ivy/demos/learn_the_basics/04_transpile_code.html) 
+- [How to write a framework-agnostic code?](https://unify.ai/docs/ivy/demos/learn_the_basics/01_write_ivy_code.html) 
+- Accelerate your development (WIP)
+- Autotune and optimize models (WIP) 
 
+------------------------------------------------------------------------
+### Ivy in 3 steps
```

Comment:
what do you mean?

Reply:
I mean to replace this section with just the installation instructions instead :)

Reply:
Furthermore, the installation instructions can maybe be the first thing in the `Getting Started` section above

Reply:
As for the getting started, we can add one or two links later.

Reply:
I think in a lot of cases it will be the other way around, the readme is not only for new people who come across the project, but also for people who know about the project and want clear information. Installation instructions are usually first class citizens in the readme I'd say

- https://github.com/pytorch/pytorch#installation
- https://github.com/tensorflow/tensorflow#install (almost the only thing they have)
- https://github.com/google/jax#installation

A lot of people will install it and then go to the examples or the docs directly imo

Reply:
Thanks, I added that!